docs/source/upgrade/ldap-migration.md

```diff
@@ -0,0 +1,241 @@
+# Overview
+
+This documentation demonstrates how to upgrade from CN 4.2 LDAP to 4.5 PostgreSQL.
+
+## Prerequisites
+
+   CN 4.2 is already installed.
+
+## Upgrading Installation
+
+### Step 1
+
+1.  Make sure the ownership of the opendj volume is `1000:1000`. For example, you can run:
+   `chown -R 1000:1000 /path/of/volume`
+1.  Backup persistence entries as the process is not reversible.
+
+1.  Backup existing `values.yaml` (used in Gluu 4.2 installation) as `values-4.2.yaml`.
+
+1.  Get new `values.yaml` for Gluu 4.5 installation.
+
+1.  Compare `values-4.2.yaml` with the new `values.yaml`, and then modify `values.yaml`:
+
+    ```yaml
+    global:
+      jackrabbit:
+        enabled: true
+      upgrade:
```

Comment:
The newest image `gluufederation/upgrade:4.5.3-2` has been released. The YAML should reflect this new image.